core/templates/dev/head/domain/question/QuestionObjectFactory.js

```diff
@@ -97,6 +99,19 @@ oppia.factory('QuestionObjectFactory', [
       );
     };
 
+    Question.prototype.toBackendDict = function(newQuestion) {
```

Comment:
Remember to add tests for all new functionality (not just this one).

Reply:
Done.